pkg/vscode/v1/Makefile

```
@@ -0,0 +1,37 @@
+# GENERATE_VSCODE_SETTINGS can be set to false so that the user can opt
+# out of generating the VSCode settings file.
+GENERATE_VSCODE_SETTINGS ?= true
+
+# VSCODE_SETTINGS_FILE is the VSCode workspace settings file that contains
+# workspace-specific settings that override user-defined settings and their
+# defaults.
+# More about VSCODE workspace settings can be found at:
+# https://code.visualstudio.com/docs/getstarted/settings#_workspace-settings
+VSCODE_SETTINGS_FILE = $(MF_PROJECT_ROOT)/.vscode/settings.json
```

Can VSCode be configured to use a different path for this file?

If not, suggest:

```suggestion
_VSCODE_SETTINGS_FILE = $(MF_PROJECT_ROOT)/.vscode/settings.json
```